src/content/docs/en/basics/astro-pages.mdx

```diff
@@ -96,10 +96,56 @@ Files with the `.html` file extension can be placed in the `src/pages/` director
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `src/pages`.
 
 This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
 
+## Custom 500 Error Page
```

Comment:
Should the `<Since>` component go here? Is this a new feature/functionality? If the whole using a custom 500 page is the new feature, then I think it goes HERE and not necessarily in the subheadings. If the feature is from 4.11, then if there are changes/additions, we can mark those with updated version numbers?

Reply:
Custom 500 pages have been supported since some time but they were not documented yet. Retrieving the error is really the new thing in 4.11 so I think the Since component is in the right place

Reply:
Gotcha! Then, this seems fine!